.github/workflows/cd.yml

```diff
@@ -10,6 +10,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - run: |
+          echo "JINAD_VERSION=$(sed -n '/^__version__/p' ./jinad/__init__.py | cut -d \' -f2)-devel" >> $GITHUB_ENV
```

Comment:
Should we just have the version in [config.py](https://github.com/jina-ai/jinad/blob/feat-add-pypi-setup/jinad/config.py#L12)?

Reply:
In the `config.py`, the version is the api version. Is it always consistent with the jinad version?

Reply:
The API version can be the same as `jinad` version

Reply:
The version is now consistent with `config.py`. The version information will stick to semantic versioning, while `JINAD_PREFIX` will denote whether it is a breaking change in the API.